src/main/java/seedu/duke/command/SwitchWorkspaceCommand.java

```diff
@@ -26,10 +25,9 @@ public SwitchWorkspaceCommand() {
     @Override
     public String execute(AnimeData animeData, StorageManager storageManager, User user) throws AniException {
         //Find the user and setActiveUser to it
-        Workspace chgWorkspace = user.getWorkspace(switchToThisWorkspace);
-        user.setActiveWorkspace(chgWorkspace);
-        String result = SWITCH_SUCCESS_HEADER + switchToThisWorkspace;
-        return result;
+        //Workspace chgWorkspace = user.getWorkspace(switchToThisWorkspace);
```

Comment:
Do you still need this? Otherwise, it would be better to remove the old code.

Reply:
Ah, I appear to have left it there mistakenly. I have fixed it, thank you for pointing it out.